tests/unit/service/conftest.py

```diff
@@ -197,6 +199,11 @@ def do_mock(
         request: httpx.Request = None,
         **kwargs,
     ) -> Response:
+        database_properties = json.loads(request.read().decode("utf-8"))["database"]
+
+        mock_database.name = database_properties["name"]
```

Comment:
I think that we can just return `database_properties` as `json` parameter in `to_response`

Reply:
the mock_database fixture set some additional default parameters, e.g. database_id. I would prefer to keep the responsibility for the default parameters to the mock_database function.

Reply:
Ok, I'm fine with this approach